docroot/modules/custom/uiowa_core/src/HeadlineHelper.php

```diff
@@ -40,8 +40,18 @@ public static function getHeadingSizeUp($size) {
   public static function getStyles() {
     return [
       'default' => 'headline block__headline',
-      'headline_bold_serif' => 'headline bold-headline bold-headline--serif block__headline',
-      'headline_bold_serif_underline' => 'headline bold-headline bold-headline--serif bold-headline--underline block__headline',
+      'headline_bold_serif' => 'headline headline--serif block__headline',
+      'headline_bold_serif_underline' => 'headline headline--serif headline--underline block__headline',
+    ];
+  }
+
+  /**
+   * Get a list of valid heading alignments.
+   */
+  public static function getHeadingAlignment() {
+    return [
+      'default' => ' ',
```

Comment:
So this `' '` causes extra spaces in the class attribute but `''` returns `default`. Is there a reason why we aren't using `headline--left.` I assume we don't have a uids class and it would just be ignored but not be spaces either. This isn't breaking anything but just something to think about I guess.

Reply:
`headline--left` didn't ever occur to me, brb need to try it. It would be great to not have extra spaces or a non-existent 'default' class.

Reply:
`headline--left` works great on new blocks and pre-existing listblocks. For some reason I'm having trouble with other pre-existing blocks, even after setting the default value in the update hook to `headline--left`.

Reply:
Added a todo for another day.